src/app/login/Login.tsx

```diff
@@ -0,0 +1,33 @@
+import Avatar from '@mui/material/Avatar';
+import CssBaseline from '@mui/material/CssBaseline';
+import Box from '@mui/material/Box';
+import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
+import Typography from '@mui/material/Typography';
+import Container from '@mui/material/Container';
+
+import { styles } from './Login.styles';
+import { useLocale } from 'hooks';
+import { LoginForm } from './loginForm/LoginForm';
+
+export const Login = () => {
+  const { formatMessage } = useLocale();
+
+  return (
+    <Container component="main" maxWidth="xs">
+      <CssBaseline />
+      <Box sx={styles.wrapper}>
+        <Avatar sx={styles.avatar}>
+          <LockOutlinedIcon />
+        </Avatar>
+        <Typography component="h1" variant="h5">
```

Comment:
Ciekawe... a `component` nie powinien od razu z wartością `h5`?

Reply:
Rzeczywiście ciekawe, wziąłem to z materiala bez zastanowienia, ale jak teraz to pozmieniałem i rozważam, dochodzę do wniosku, że ma to sens - `component="h1"` ma sens pod względem SEO - tekst z h1 to tytuł strony. Jednak jeżeli mu zostawimy style domyślne h1 to jest bardzo mały, dlatego mamy tu `variant="h5"`